plugin/src/lib.rs

```diff
@@ -724,6 +724,23 @@ fn fold_first_block(block: P<Block>, m: &mut MutatorPlugin) -> P<Block> {
                         .unwrap(),
                 );
             }
+            let (ref key, ref values) = for (key, &value) in interchangeables.iter() {
```

Comment:
that line should probably read `for (ref key, ref values) in &interchangesables {`.
Please note that this means key is a `&Symbol` and values is a `&Vec<Symbol>`, so we also need an inner `for value in values {` loop here.